list/list.go

```diff
@@ -606,7 +607,7 @@ func (m *Model) updateKeybindings() {
 		m.KeyMap.CloseFullHelp.SetEnabled(false)
 
 	default:
-		hasItems := m.items != nil
+		hasItems := len(m.items) != 0
```

Comment:
These two statements are actually identical. I believe in this library we’re using the shorter convention.

Reply:
It's not identical. A nil slice is not the same thing as a slice with zero elements. The zero-element slice will return true for the != nil check.

Reply:
You can see a proof of this here:

https://go.dev/play/p/0hlHLaxSACV

Reply:
Good call: I stand corrected